src/pages/patientView/vafPlot/ThumbnailExpandVAFPlot.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import Tooltip from 'rc-tooltip';
+import 'rc-tooltip/assets/bootstrap_white.css';
+import {VAFPlot, IVAFPlotProps} from './VAFPlot';
+
+export type IThumbnailExpandVAFPlotProps = {
+    data?: typeof IVAFPlotProps['data'];
+    order?: typeof IVAFPlotProps['order'];
+    colors?: typeof IVAFPlotProps['colors'];
+    labels?: typeof IVAFPlotProps['labels'];
+    overlayPlacement?: string;
+};
+
+export class ThumbnailExpandVAFPlot extends React.Component<IThumbnailExpandVAFPlotProps, {}> {
+
+    shouldComponentUpdate() {
+        return false;
+    }
+
+    render() {
+        let thumbnailProps = {
+            data: this.props.data,
+            colors: this.props.colors,
+            order: this.props.order,
+            show_controls: false,
+            nolegend: true,
+            width: 64,
+            height: 64,
+            label_font_size: "6.5px",
+            xticks: 0,
+            yticks: 0,
+            margin_bottom: 15,
+        };
+        let expandedProps = {
+            data: this.props.data,
+            colors: this.props.colors,
+            labels: this.props.labels,
+            order: this.props.order,
+            show_controls: true,
+            nolegend: false,
+            init_show_histogram: true,
+            init_show_curve: true,
+        };
+
+        return (<span>
+            <Tooltip placement={this.props.overlayPlacement} trigger={['hover', 'focus']} overlay={<VAFPlot {...expandedProps}/>}
+                    arrowContent={<div className="rc-tooltip-arrow-inner"></div>}
+                    destroyTooltipOnHide={false}>
```

Comment:
i think destroy:true is generally a good idea unless there's some big cost in reproducing the tooltip